tests/stress_tests/common/utils.h

```diff
@@ -9,12 +9,12 @@
 #include <vector>
 #include <thread>
 #include <functional>
-#include <sys/unistd.h>
-#include <sys/wait.h>
 
 #ifdef _WIN32
 #define OS_SEP std::string("\\")
 #else
+#include <sys/unistd.h>
+#include <sys/wait.h>
```

Comment:
OS specific headers in .h

Reply:
Deleted it in header file and transferred implementation run_in_process to cpp

Reply:
Unfortunately, I can't remove OS specific headers in .h, because some methods using in template function